src/avatar.tsx

```diff
@@ -99,13 +80,20 @@ const Avatar = ({
 
     return (
         <Badge value={badgeValue} top={badgePosition} right={badgePosition}>
-            <div className={classes.avatar} role="img" aria-label={ariaLabel ?? initials}>
+            <div
+                className={classes.avatar}
+                role="img"
+                aria-label={ariaLabel ?? initials}
+                style={{width: size, height: size, color: textColor, background: backgroundColor}}
```

Comment:
so now we will use more and more inline styles to support things like 'isInverse' colors right? or could we create a common set of styles with sprinkles for that?

Reply:
we can use sprinkles when possible (as an alternative to inline styles), but only when dynamic values are available in our sprinkles definitions. For example, if a component receives a prop `color`, we can use it with sprinkles if the color is one of the skin, but if the `color` prop accepts any string then we need to use inline styles